### ISSUE TITLE
Fog renders wrong on water

### ISSUE THREAD

**Issue by SurDno** [Decane/SRP#74]
![image](https://user-images.githubusercontent.com/35204405/48681672-e9907800-ebb4-11e8-9cd9-fabfb5cebeaa.png)

**Comment by Decane** [comment]
Are you using a weather mod? I don't think vanilla CS (or CS with the SRP) has fog as thick as that. Regardless, I assent that this is a vanilla (shader) bug. It also manifests with vanilla fog.

**Comment by SurDno** [comment]
Yep, Atmosfear. It may be less noticeable in vanilla, but it still exists.

**Comment by SH1F7ER** [comment]
Btw soft water isn't working on radioactive puddles at least in Dark Valley and Garbage. I guess the developers forgot to add some kind of flag which indicates that the puddles are actually water
